Wurlitzer.h

```diff
@@ -0,0 +1,26 @@
+struct Wurlitzer
```

Comment:
Where is the `#pragma once`?
Where are the `#include` statements for the non-primitive types and macros?
- [ ] revise